app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -57,6 +59,9 @@ export default function ResourceAuthoringPage() {
         }
       }
     }
+    if (resource?.resourceType === 'Measure' && library !== resource?.library?.[0]) {
```

Comment:
Thoughts on moving this into the the following return statement boolean? Or do you think it's more readable here?

Reply:
Regardless of where it is, I do think the logic here has to be ever so slightly altered. Super specific situation, but if the user removes the library and clicks submit, then the submit button stays enabled. The submit button will stay enabled regardless of changes if the library is not defined (which I don't think is the behavior we want?).

Reply:
I think it's because `library` is null and the library ON the resource is `undefined`...maybe changing the type of `library` would fix this?